src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -2846,7 +2846,8 @@ namespace SurfaceGeometry {
 
         for (int surfNum = 1; surfNum <= state.dataSurface->TotSurfaces; ++surfNum) {
             auto &thisSurf = Surfaces(surfNum);
-            if (!thisSurf.HeatTransSurf) continue; // ignore shading surfaces
+            if (!thisSurf.HeatTransSurf) continue;                               // ignore shading surfaces
+            if (thisSurf.Class == DataSurfaces::SurfaceClass::IntMass) continue; // skip internal mass surfaces for this check
```

Comment:
When checking the surface space assignments, don't want a stray InternalMass object to trigger adding a new -Remainder space.

Reply:
I don't really understand this, but it does seem to make sense.